modules/preview-react/pill/lib/Pill.Avatar.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import {createComponent} from '@workday/canvas-kit-react/common';
+import {Avatar, AvatarProps} from '@workday/canvas-kit-react/avatar';
+
+export interface PillAvatarProps extends AvatarProps {}
+
+export const PillAvatar = createComponent('button')({
+  displayName: 'Pill.Avatar',
+  Component: ({children, ...elemProps}: PillAvatarProps, ref, Element) => {
+    return (
+      <Avatar as="span" size={20} ref={ref} {...elemProps}>
```

Comment:
Hmm maybe I'm not understanding the overload avatar: https://github.com/Workday/canvas-kit/blob/master/modules/react/avatar/lib/Avatar.tsx#L49

If I don't pass an `as` prop it default to a button. I wanted to avoid them setting this as any other element since the avatar already sits inside a `button` element, thoughts?